spark-jobs/pom.xml

```diff
@@ -181,9 +179,9 @@ spark-submit -(remove this)-packages org.apache.spark:spark-sql-kafka-0-10_2.11:
 -->
                             <exclude>org.apache.hadoop:*</exclude>
                             <exclude>org.scala-lang:scala-actors</exclude>
-                            <exclude>org.scala-lang:scala-compiler</exclude>
-                            <exclude>org.scala-lang:scala-reflect</exclude>
-                            <exclude>org.scala-lang:scala-library</exclude>
+<!--                            <exclude>org.scala-lang:scala-compiler</exclude>-->
```

Comment:
Do you know why this is present as commented out. And not removed fully?

Reply:
I think it's leftover code from before. It shouldn't be commented out